chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/QuestionableActivityEmailJob.java

```diff
@@ -73,9 +73,14 @@
     private static final int ACTIVITY_CERT_STATUS_CHANGE_REASON_COL = 11;
     private static final int ACTIVITY_REASON_COL = 12;
 
+    private static final int MIN_RANGE_IN_DAYS = 1;
```

Comment:
Is there any way to pull these values from the jobs.xml definition? If we ever change them (for whatever reason) we'd have to change them in two places. I'm also thinking about what happens with other jobs that we might add range values for; it'd be nice to have a paradigm for having those values defined in only one place